features/docs/cli/publish_banner.feature

```diff
@@ -0,0 +1,37 @@
+Feature: Publish banner
+
+  A banner is displayed on stderr at the end of a run to inform that reports can
+  be published on reports.cucumber.io[reports.cucumber.io](https://reports.cucumber.io).
+
+  Scenario: Banner is displayed when running Cucumber
+    Given a file named "features/hello.feature" with:
+    """
+    Feature: Hello
+    """
+    When I run `cucumber`
+    Then the output should contain:
+    """
+    ┌──────────────────────────────────────────────────────────────────────────┐
+    │ Share your Cucumber Report with your team at https://reports.cucumber.io │
+    │                                                                          │
+    │ Command line option:    --publish                                        │
+    │ Environment variable:   CUCUMBER_PLUGIN_PUBLISH_ENABLED=true             │
+    │                                                                          │
```

Comment:
Add a 3rd option

```
cucumber.yml:      default: --publish
```

I know it's similar to option 1, but it might help with adoption.

Reply:
done